### ISSUE TITLE
Terrain data not saving in scenario editor.

### ISSUE THREAD

**Issue by lucusloc** [OpenRCT2/OpenRCT2#14750]
**OS:** Windows 10
**Version:** 0.3.3
**Commit/Build:** Downloaded ~10 PM 5/26

Scenarios not saving properly.
Scenario not available on start game list. Scenario file is visible in scenario folder, but when opened terrain is flat. Object selection also not saved.

- [ ] Reproducible in RCT2 (vanilla)?
no
- [ ] Specific to multiplayer?
- [ ]  no

**Steps to reproduce:**
1. Create a new scenario. 
2. Load all objects (rides, decorations, etc.). 
3. Edit terrain (used random generator), park size 256. 
4. Set park entrances and guest spawn (one per side). 
5. Add paths and decorations. Set scenario parameters (no money, have fun). 
6. Save scenario. 
7. Attempt to open scenario in new game or in scenario editor again. 

**Dump file**

[Easy Lakes Beta.park.sc6.zip](https://github.com/OpenRCT2/OpenRCT2/files/6544509/Easy.Lakes.Beta.park.sc6.zip)
[Easy Test Park.park.sc6.zip](https://github.com/OpenRCT2/OpenRCT2/files/6544512/Easy.Test.Park.park.sc6.zip)

**Screenshots / Video:**

**Save game:**

**Comment by Gymnasiast** [comment]
Please fill in the build properly. It's listed in the title screen and looks something like ab1c2d3e.

**Comment by lucusloc** [comment]
Found the version; 16c10f6 (I think, font is a little off). The x86-64 version.

**Comment by Gymnasiast** [comment]
So you’re on the new-save-format test branch. Should have said that :/

**Comment by lucusloc** [comment]
Yes, I though this space was only for the new save beta? I did not mean to report this to the general OpenRCT2 branch.

**Comment by Gymnasiast** [comment]
This space is called OpenRCT2/OpenRCT2. Not sure why you thought it was specifically for the new-save-format test builds.

Anyway, it is used for both and if you use a test build for the new save format, you need to explicitly mention it. This also underlines the need for specifying the commit - it gives us a lot of information.

Back to the issue at hand. How did you save your scenario? Did you use the button in the lower right corner of the scenario editor, or did you use `Save Landscape`?

**Comment by lucusloc** [comment]
I was attempting to make a full scenario, so I used the one at the bottom of the screen after walking through all the steps (select objects, edit terrain, set up research, set up scenario objectives, save). Worked through it twice, once from a previously saved scenario (from the release version, so old save format), and once from scratch to verify it was an issue with the save function itself and not an artifact of the file conversion. 

I was not able to get video capture working through the game bar, but I can do a screen share if that would be helpful.

**Comment by Gymnasiast** [comment]
We got another report, so I’m not sure if this can be closed.
